.travis-build.sh

```diff
@@ -88,14 +72,15 @@ if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
   else
     CFISLUGOWNER=${SLUGOWNER}
   fi
+  CFISLUGOWNER=smillst
   set -e
   echo "Running:  (cd .. && git clone --depth 1 https://github.com/${CFISLUGOWNER}/checker-framework-inference.git)"
-  (cd .. && git clone --depth 1 https://github.com/${CFISLUGOWNER}/checker-framework-inference.git) || (cd .. && git clone --depth 1 https://github.com/${CFISLUGOWNER}/checker-framework-inference.git)
+  (cd .. && git clone --depth 1 -b gradle https://github.com/${CFISLUGOWNER}/checker-framework-inference.git) || (cd .. && git clone -b gradle --depth 1 https://github.com/${CFISLUGOWNER}/checker-framework-inference.git)
```

Comment:
Note: This needs to be reverted before merge.  (See PR https://github.com/typetools/checker-framework-inference/pull/71)

Reply:
I merged that PR, so the change above can be undone.